SportsHub/SportsHub.Core/SportsHub.Domain.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2-beta2" />
```

Comment:
Why do you need this package? Is it possible to use `System.Text.Json`?

Reply:
Yes it is possible to use System.Text.Json , but Microsoft have not implemented all of the features of Newtonsoft . That is the reason why with System.Text.Json in some scenarios you need to make custom converter for them . From my point of view and my knowledge it was easier for me to use Newtonsoft.Json and serialize the object this way

Reply:
You are right some features are not implemented in `System.Text.Json`  but nevertheless, it can cover a lot of required functionality. As for me, I prefer not to drag one more package into the project. Let's try `System.Text.Json`  and if we see that it lacks some functionality, we will discuss what to do to implement it or add `Newtonsoft`.

Reply:
If you are not using JSON serialization, then the is no reason to add `Newtonsoft` at all.

Reply:
I wont use it